### ISSUE TITLE
Initial display incorrect with international phone number

### ISSUE THREAD

**Issue by ajmas** [CdTgr/vue3-q-tel-input#4]
If I provide an initial number such '3227329721', then '+1' is displayed by default:

<img width="683" alt="Screenshot 2021-11-27 at 23 54 14" src="https://user-images.githubusercontent.com/682269/143730124-93737516-fbb6-437a-b1e1-c2ab0c784b09.png">

Expectations is that the drop down should contain the correct prefix, while the rest of the number should be displayed outside of the field.

As used in Vue:

```html
<vue3-q-tel-input v-model:tel="phoneNumber" outlined class="q-ma-xs"/>
```

**Comment by kiatng** [comment]
+1 for this issue.

**Comment by CdTgr** [comment]
@ajmas I didn't understand the issue well. Could you please explain it better in a video?

**Comment by apurvfortius** [comment]
default contry passed in props. but it did not get selected by default until element get focused.

**Comment by Tofandel** [comment]
Same issue
```vue
          <Vue3QTelInput
            v-model:tel="phone"
            default-country="fr"
          />
```

Will show `+1` and `+33` only when focused

**Comment by ajmas** [comment]
I haven’t touched this in a bit. I’ll see if I can make a test case

**Comment by MHassanSaqib** [comment]
any solution

**Comment by CdTgr** [comment]
Should be fixed in [1.0.25](https://www.npmjs.com/package/vue3-q-tel-input/v/1.0.25)
